heudiconv/dicoms.py

```diff
@@ -62,6 +62,14 @@ def create_seqinfo(mw, series_files, series_id):
     global total_files
     total_files += len(series_files)
 
+    custom_seqinfo_data = custom_seqinfo(wrapper=mw, series_files=series_files) \
```

Comment:
```suggestion
    custom_seqinfo_data = custom_seqinfo(series_files=series_files, wrapper=mw) \
```
to correspond order in the doc?